api/v1/words/getById.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+module.exports = (request, reply) => {
+  const r = request.server.plugins['hapi-rethinkdb'].rethinkdb;
+  // r === this.rethinkdb;
+
+  const conn = request.server.plugins['hapi-rethinkdb'].connection;
+  // conn === this.rethinkdbConn;
+  
+  const wordId = request.params.id;
+  
+   r.table('words')
+    .get(wordId)
+    .run(conn)
+    .then(
+      function (userResponse) {
+        reply(userResponse);
```

Comment:
Would like to make few things clearer. The problem that we have throughout wordist API is that , we are not sending back response with correct status code. Lets use https://www.npmjs.com/package/hapi-status to return 404 if the word is not found.